### PR TITLE
added transition effect to model

### PR DESCRIPTION
**Pull request by gmrzone**
Added transition effect to model

### CODE REVIEW

**Comment by Mr-spShuvo** [comment]
Has conflicts. Can you resolve them?

**Comment by gmrzone** [comment]
yes

File: frontend/src/Pages/UserSettingsPage.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ import '../styles/components/UserSettings.css';
 import { useState } from 'react';
 import { Link } from 'react-router-dom';
 import User from '../data/users';
-import UserSettingUpdateModel from '../Components/UserSettingUpdateModel';
+import UserSettingUpdateModel from '../Components/UserSettingUpdateModal';
```
Comment (Mr-spShuvo) [inline frontend/src/Pages/UserSettingsPage.js:6]: Naming mitchmatch
`import UserSettingUpdateModel from '../Components/UserSettingUpdateModal'`

File: frontend/src/Components/UserSettingUpdateModal.js [hunk 2]
```diff
@@ -76,7 +77,7 @@ const UserSettingUpdateModel = ({
       return (
         <>
           <div className="form__field">
-            <label htmlFor="formInput#text">User Name: </label>
+            <label htmlFor="formInput#text">UserName: </label>
```
Comment (Mr-spShuvo) [inline frontend/src/Components/UserSettingUpdateModal.js:80]: Make it, `Username`

File: frontend/src/Components/UserSettingUpdateModal.js [hunk 3]
```diff
@@ -59,7 +60,7 @@ const UserSettingUpdateModel = ({
             />
           </div>
           <div className="form__field">
-            <label htmlFor="formInput#textarea">Message: </label>
+            <label htmlFor="formInput#textarea">Bio: </label>
```
Comment (Mr-spShuvo) [inline frontend/src/Components/UserSettingUpdateModal.js:63]: Make it `About`

**Review by Mr-spShuvo — CHANGES_REQUESTED** [review]
Make those changes

File: frontend/src/Components/Modal.js [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+  };
+  const toggleOff1 = () => {
+    if (modalRef.current) {
+      modalRef.current.style.display = 'none';
+    }
+  };
+  return ReactDom.createPortal(
+    <div className={`modal-backdrop`} ref={modalRef}>
+      <div style={{ display: 'none' }}>
+        {active ? toggleOn() : toggleOff()}
```
Comment (codyseibert) [inline frontend/src/Components/Modal.js:29]: Do you think using a react animation library such as Framer Motion could help prevent needing to add setTimeouts inside the code and directly change inline styles?

**Review by Mr-spShuvo — APPROVED** [review]

**Pull request merged**